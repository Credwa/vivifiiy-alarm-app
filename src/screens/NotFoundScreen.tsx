import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import { SafeAreaView } from 'react-native-safe-area-context';

export default function NotFoundTabScreen() {
  return (
    <SafeAreaView style={styles.container}>
      <Text style={styles.title}>Not found</Text>
      <View style={styles.separator} />
    </SafeAreaView>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center'
  },
  title: {
    fontSize: 20,
    fontWeight: 'bold'
  },
  separator: {
    marginVertical: 30,
    height: 1,
    width: '80%'
  }
});
