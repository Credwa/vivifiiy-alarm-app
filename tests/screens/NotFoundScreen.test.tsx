import NotFoundScreen from '@/screens/NotFoundScreen.tsx';
import { Ionicons } from '@expo/vector-icons';
import * as Font from 'expo-font';
import React from 'react';
import renderer from 'react-test-renderer';

describe('<NotFoundScreen />', () => {
  beforeEach(async () => {
    await Font.loadAsync({
      ...Ionicons.font
    });
  });

  it(`renders correctly`, () => {
    const tree = renderer.create(<NotFoundScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
