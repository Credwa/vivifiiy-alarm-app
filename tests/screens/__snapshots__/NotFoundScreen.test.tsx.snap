// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundScreen /> renders correctly 1`] = `
<RNCSafeAreaView
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    Not found
  </Text>
  <View
    style={
      Object {
        "height": 1,
        "marginVertical": 30,
        "width": "80%",
      }
    }
  />
</RNCSafeAreaView>
`;
